server/build_event_protocol/accumulator/accumulator.go

```diff
@@ -127,6 +129,17 @@ func (v *BEValues) AddEvent(event *build_event_stream.BuildEvent) {
 				break
 			}
 		}
+	case *build_event_stream.BuildEvent_TestResult:
+		for _, f := range p.TestResult.TestActionOutput {
+			if f.GetName() == "test.log" {
+				if url, err := url.Parse(f.GetUri()); err != nil {
+					log.Warningf("Error parsing uri from TestResult: %s", f.GetUri())
+				} else {
+					v.testLogURI = url
+					v.testLogName = f.GetName()
+				}
+			}
+		}
```

Comment:
If there are multiple test results, then I think we'll want the ability to serve multiple log files.

We don't want to add multiple log files to the accumulator though since those are variable in length and in general we want the accumulator to be a flat struct that doesn't depend on the size of the invocation, since it sits in memory while the invocation is in progress.

Instead, we could have the test log endpoint read the invocation and then scan the BES events for the desired test log.

Reply:
Done!